src/dnmd/CMakeLists.txt

```diff
@@ -18,11 +18,18 @@ add_library(dnmd
   ${HEADERS}
 )
 
-target_include_directories(dnmd PUBLIC ../inc)
-target_include_directories(dnmd PUBLIC ../inc/external) # Hiding the "external" subdirectory due to uses of <...> in cor.h.
+add_library(dnmd::dnmd ALIAS dnmd)
+
+target_include_directories(dnmd PUBLIC $<INSTALL_INTERFACE:include>)
 
 set_target_properties(dnmd PROPERTIES
-  PUBLIC_HEADER ../inc/dnmd.h
-  PUBLIC_HEADER ../inc/dnmd.hpp)
+  PUBLIC_HEADER "../inc/dnmd.h;../inc/dnmd.hpp")
```

Comment:
My assumption here was that `dnmd.hpp` would only be included when someone references `dnmd_interfaces`. Do we want them to have access to the C++ header if they are just accessing `dnmd`?

Reply:
`dnmd.hpp` provides the `mdhandle_ptr` type, which I feel is decently useful for any C++-based user of DNMD.

For `dnmd_interfaces`, there's the `dnmd_interfaces.hpp` header for any users of the interfaces library (which doesn't actually expose any DNMD APIs through its surface area).

Maybe we should move `malloc_ptr` from `dnmd.hpp` to one of the internal headers (like `dnmd_platform.hpp`) since it's only for internal usage?

Reply:
Oh right. You're right. I conflated them. Thanks!

Reply:
> Maybe we should move malloc_ptr from dnmd.hpp to one of the internal headers (like dnmd_platform.hpp) since it's only for internal usage?

@jkoritzinsky That does make sense.